master/security/service-accounts.md

```diff
@@ -87,6 +87,29 @@ spec:
   selector: 'app == "db"'
 ```
 
+#### Restrict label assignment with service account selectors
```

Comment:
Probably need to add a bullet point here for this new section: https://github.com/projectcalico/calico/pull/2911/files#diff-ad42278cb872bb083fb60cb21c746d85R48

Also, I might tweak the wording of this title. What about "Use Kubernetes RBAC to control service account label assignment"?

Reply:
Sure